lib/checkPullRequestLabels.js

```diff
@@ -15,7 +15,8 @@ module.exports.checkAssignee = async function(context) {
     pullRequestNumber + ' ...');
 
   var changelogLabel = context.payload.label.name.trim();
-  var hasChangelogLabel = changelogLabel.toUpperCase().startsWith('CHANGELOG');
+  var hasChangelogLabel = changelogLabel.toUpperCase().startsWith(
+    'PR CHANGELOG');
```

Comment:
Let's rename this to `PR: CHANGELOG` to be consistent with other PR labels.

Reply:
^ mind the colon

Reply:
Replied here: https://github.com/oppia/oppia/pull/8515#discussion_r372218867